### PR TITLE
Ja localization

### PR DESCRIPTION
**Pull request by Mayuko-Sa**
# Description

Updated EN -> JA documentation translations
This translation is based on [746f7b2](https://github.com/circleci/circleci-docs/commit/746f7b29bb1f5c0e131bcd68fb69ad6534f4d394)

### CODE REVIEW

File: jekyll/_cci2_ja/runner-installation-linux.adoc [hunk 1]
```diff
@@ -246,3 +246,8 @@ circleci.service - CircleCI Runner
 ```shell
 journalctl -u circleci
 ```
+
+[#troubleshooting]
+== トラブルシューティング
+
+Refer to the <<troubleshoot-self-hosted-runner#troubleshoot-machine-runner, Troubleshoot Machine Runner section>> of the Troubleshoot Self-hosted Runner guide if you encounter issues installing or running machine runner on Linux.
```
Comment (kbatuigas) [inline jekyll/_cci2_ja/runner-installation-linux.adoc:253]: Hello, @Mayuko-Sa ! Do we expect this text to also be translated within this PR?

File: jekyll/_cci2_ja/runner-installation-mac.adoc [hunk 2]
```diff
@@ -174,3 +174,8 @@ sudo launchctl unload '/Library/LaunchDaemons/com.circleci.runner.plist'
 == サービスの実行状態を確認する
 
 プリインストールされている macOS アプリケーション、 *コンソール* を開きます。 このアプリケーションでは、 *ログレポート* で CircleCI エージェントのログを閲覧することができます。 リストから `com.circleci.runner.log` というログを探します。 このファイルは、 *Library > Logs* に移動しても見つけられます。
+
+[#troubleshooting]
+== トラブルシューティング
+
+Refer to the <<troubleshoot-self-hosted-runner#troubleshoot-machine-runner, Troubleshoot Machine Runner section>> of the Troubleshoot Self-hosted Runner guide if you encounter issues installing or running machine runner on macOS.
```
Comment (kbatuigas) [inline jekyll/_cci2_ja/runner-installation-mac.adoc:181]: @Mayuko-Sa Same question as above

**Comment by Mayuko-Sa** [comment]
@kbatuigas Thank you!! Somehow Crowdin didn't push the translation 😅

**Review by kbatuigas — APPROVED** [review]
Thank you! 🎉

**Review by saori-circleci — APPROVED** [review]

**Pull request merged**